docs/how_to/visualize_a_quantum_circuit.rst

```diff
@@ -0,0 +1,140 @@
+------------------------------
+
+If you call the ``print()`` function on a :class:`~.QuantumCircuit` object, you will get an `ASCII art version <https://en.wikipedia.org/wiki/ASCII_art>`_ of the circuit diagram.
+
+.. testcode::
+
+    print(qc)
+
+.. testoutput::
+    :options: +NORMALIZE_WHITESPACE
+
+         ┌───┐     ┌─┐   
+    q_0: ┤ H ├──■──┤M├───
+         ├───┤┌─┴─┐└╥┘┌─┐
+    q_1: ┤ H ├┤ X ├─╫─┤M├
+         ├───┤└┬─┬┘ ║ └╥┘
+    q_2: ┤ H ├─┤M├──╫──╫─
+         └───┘ └╥┘  ║  ║ 
+    c: 3/═══════╩═══╩══╩═
+                2   0  1 
+
+Using the :meth:`~.QuantumCircuit.draw` method
+----------------------------------------------
+
+You can also use the :meth:`.QuantumCircuit.draw` method to visualize it. The default output style is 'text', which will output a ASCII art version, the same as using the ``print()`` function.
+
+.. code-block:: python
+
+    qc.draw()
+
+.. code-block:: text
+
+         ┌───┐     ┌─┐   
+    q_0: ┤ H ├──■──┤M├───
+         ├───┤┌─┴─┐└╥┘┌─┐
+    q_1: ┤ H ├┤ X ├─╫─┤M├
+         ├───┤└┬─┬┘ ║ └╥┘
+    q_2: ┤ H ├─┤M├──╫──╫─
+         └───┘ └╥┘  ║  ║ 
+    c: 3/═══════╩═══╩══╩═
+                2   0  1  
+
+You can also change the output style using the ``output`` argument. These are the available output styles:
+
+1. ``'text'``: renders the circuit with ASCII art. It's the default option.
+2. ``'mpl'``: uses `matplotlib <https://matplotlib.org/>`_ to render the circuit.
+3. ``'latex'``: uses :math:`\LaTeX` to render the circuit. It requires a full `LaTeX <https://latex.org/forum/>`_ distribution and the package ``pdflatex``.
+4. ``'latex_source'``: outputs the :math:`\LaTeX` source code that creates the ``'latex'`` rendering of the circuit.
+
+Because this optional keyword argument is the first of this method, you can type ``qc.draw(option)`` instead of ``qc.draw(output=option)``
+
+.. note::
+    By default, the :meth:`~.QuantumCircuit.draw` method returns the rendered image as an object and does not output anything. The exact class returned depends on the output specified: ``'text'`` (the default) returns a ``TextDrawer`` object, ``'mpl'`` returns a `matplotlib.figure.Figure <https://matplotlib.org/stable/api/figure_api.html?highlight=figure#matplotlib.figure.Figure>`_ object, and ``'latex'`` returns a ``PIL.Image`` object. Having the return types enables modifying or directly interacting with the rendered output from the drawers. Jupyter notebooks understand these return types and render them for us in this guide, but when running outside of Jupyter, you do not have this feature automatically. However, the :meth:`~.QuantumCircuit.draw` method has optional arguments to display or save the output. When specified, the ``filename`` kwarg takes a path to which it saves the rendered output. Alternatively, if you're using the ``'mpl'`` or ``'latex'`` outputs, you can leverage the ``interactive`` kwarg to open the image in a new window (this will not always work from within a notebook).
```

Comment:
```suggestion
    By default, the :meth:`~.QuantumCircuit.draw` method returns the rendered image as an object and does not output anything. The exact class returned depends on the output specified: ``'text'`` (the default) returns a ``TextDrawer`` object, ``'mpl'`` returns a `matplotlib.figure.Figure <https://matplotlib.org/stable/api/figure_api.html?highlight=figure#matplotlib.figure.Figure>`_ object, and ``'latex'`` returns a ``PIL.Image`` object. Having these return types enables modification or direct interaction with the rendered output from the drawers. Jupyter notebooks understand these return types and render them for us in this guide, but when running outside of Jupyter, you do not have this feature automatically enabled. However, the :meth:`~.QuantumCircuit.draw` method has optional arguments to display or save the output. When specified, the ``filename`` kwarg takes a path to which it saves the rendered output. Alternatively, if you're using the ``'mpl'`` or ``'latex'`` outputs, you can leverage the ``interactive`` kwarg to open the image in a new window (this will not always work from within a notebook).
```

Reply:
tbh this paragraph is quite lengthy and I'm not sure its info that is explicitly necessary to the user in a how to guide (feels more like an explanation?). It think the most valuable bit of this is the last sentence explaining how to use the save and interactive stuff. Maybe cut this paragprah down to just that last sentence?